.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,45 @@
+---
+name: Bug report
+about: Please do NOT file bugs without filling in this form.
+title: '[Bug] '
+labels: ''
+assignees: ''
+
+---
+
+**Which version of Microsoft.IdentityModel are you using?**
+Note that to get help, you need to run the latest version. 
+<!-- E.g. Microsoft.IdentityModel 6.14 -->
+
+**Where is the issue?**
+ * [ ] M.IM.JsonWebTokens
+ * [ ] M.IM.Protocols.OpenIdConnect
+ * [ ] M.IM.Tokens
+ * [ ] M.IM.Tokens.Jwt
+ * [ ] M.IM.Validators
+* Other (please describe)
```

Comment:
nit: spacing on '*' looks a bit off, perhaps continue the [ ] syntax here too for consistency?

Reply:
might be a good idea, yes. lol.

Reply:
might want to add a check for each of our assemblies.
M.IM.Logging, M.IM.Protocols, M.IM.P.OpenIdConnect, M.IM.P.SignedHttpRequest, M.IM.P.WsFederation, M.IM.Tokens.Saml, M.IM.Xml

Note it is S.IM.Tokens.Jwt not M.IM.Tokens.Jwt

Reply:
fixed.